samples/apps/bookinfo/route-rule-all-v1.yaml

```diff
@@ -1,10 +1,35 @@
 type: route-rule
+name: productpage-default
+spec:
+  destination: productpage.default.svc.cluster.local
+  precedence: 1
+  route:
+  - tags:
+      version: v1
+---
+type: route-rule
 name: reviews-default
 spec:
   destination: reviews.default.svc.cluster.local
   precedence: 1
   route:
   - tags:
       version: v1
-    weight: 100
 ---
```

Comment:
May be make the weights explicit to let people know that total weights must add up to 100?